doc/user/content/cloud/get-started.md

```diff
@@ -13,23 +15,31 @@ This guide walks you through getting started with Materialize Cloud, from settin
 
 * Signing up for Materialize Cloud
 
-* Creating, and connecting to a Materialize Cloud deployment
+* Creating and connecting to a Materialize Cloud deployment
 
 * Connecting to a streaming data source
 
 * Creating a materialized view
 
 * Exploring common patterns like joins and time-windowing
 
-## Sign up, deploy, connect
+## Sign up and deploy
 
 1. Sign up for Materialize Cloud at [https://cloud.materialize.com](https://cloud.materialize.com#signup).
 
-1. Once you [log in](https://cloud.materialize.com), you can use the [Deployments](https://cloud.materialize.com/deployments) page to create, upgrade, or destroy deployments, as well as to obtain the TLS certificates you need to install on your local machine to connect.
+    You can create a Materialize Cloud account or sign in with an existing Google or Github acccount. SSO and 2FA are also available for [enterprise accounts](account-limits).
```

Comment:
@msudberg 

re: SSO - per @benesch comment https://github.com/MaterializeInc/cloud/issues/700#issuecomment-912213494, SSO is expected to now be out of scope for the go-live for Open Beta. Please will you ensure that documentation reflects this change accordingly.

Reply:
So, AIUI, I think the signup form at cloud.materialize.com is going to stay disabled until self-service beta? From what I understood from @craig-materialize, there's a signup flow that marketing has built out that ultimately drops your email into a sales queue, and then someone on the sales team needs to actually unblock the request with an invitation to Materialize Cloud.

Reply:
This is news to me - no one's approached me to tell me 'this is what we're going to do for the signups'. I'm open to this but let's please discuss and ensure we're on the same page on this today @craig-materialize @benesch. Will ping you on Slack shortly on this. Thank you.

Reply:
@msudberg update for you on this: @benesch @TonyRippy and I synced on this in the office today, and we have decided that due to concerns from Engineering about the scalability limits of the system and the bandwidth of the team to handle (urgent) inbound requests from users that we will be implementing an in-between step as described by Nikhil above for the signup flow. That is, users will have to get on a call with us in order to be allowed to proceed to completing the new account creation flow.

cc: @craig-materialize